### PR TITLE
Correctly escape the footer description from theming

### LINKED ISSUE #3834: Welcome panel does not parse user description
### Steps to reproduce
1. Enter HTML `<a href` link into your webserver description.
2. Show the welcome panel.

### Expected behaviour
Just as this kind of "wanted-XSS" is done in the server description on the bottom of the (login) site, also parse the HTML here.

### Actual behaviour
There the HTML seems to be escaped.

### Server configuration
**Operating system**: Debian

**Nextcloud version:** 11.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh

### Client configuration
**Browser:** Firefox 52

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #3834 
Gallery https://github.com/nextcloud/gallery/issues/222

### CODE REVIEW

**Review by LukasReschke — APPROVED** [review]
LGTM

**Review by MorrisJobke — COMMENTED** [review]
:-1:

bildschirmfoto 2017-03-16 um 15 02 00

**Comment by nickvergessen** [comment]
Fixed

**Review by MorrisJobke — APPROVED** [review]

File: apps/theming/lib/ThemingDefaults.php [hunk 1]
```diff
@@ -81,27 +81,27 @@ public function __construct(IConfig $config,
...
+		return $this->getName();
 	}
 
 	public function getBaseUrl() {
 		return $this->config->getAppValue('theming', 'url', $this->url);
 	}
 
 	public function getSlogan() {
-		return $this->config->getAppValue('theming', 'slogan', $this->slogan);
+		return \OC_Util::sanitizeHTML($this->config->getAppValue('theming', 'slogan', $this->slogan));
```
Comment (MorrisJobke) [inline apps/theming/lib/ThemingDefaults.php]:
```
line  104: OC_Util - Static method of private class must not be called
```

**Comment by MorrisJobke** [comment]
Changes look good :+1:

**Pull request merged**